content/references/protocol-reference/transactions/transaction-types/xchainaccountcreatecommit.md

```diff
@@ -0,0 +1,65 @@
+---
+html: xchainaccountcreatecommit.html 
+parent: transaction-types.html
+blurb: Create a bridge between two chains.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# XChainAccountCreateCommit
+
+The `XChainAccountCreateCommit` transaction defines a new cross-chain bridge entrance on one of the chains that the bridge connects. It includes information about the type of tokens being exchanged. To fully set up a bridge, this transaction must be executed on both chains, alongside setting up witness servers.
+
+The complete production-grade setup would also include a `SignerListSet` transaction on the two door accounts for the witnesses’ signing keys, as well as disabling the door accounts’ master key. This would ensure that the funds are truly in control of the witness servers.
+
+## Example {{currentpage.name}} JSON
+
+
+```json
+{
+  "Account": "rwEqJ2UaQHe7jihxGqmx6J4xdbGiiyMaGa",
+  "Destination": "rD323VyRjgzzhY4bFpo44rmyh2neB5d8Mo",
+  "TransactionType": "XChainAccountCreateCommit",
+  "Amount": "20000000",
+  "SignatureReward": "100",
+  "XChainBridge": {
+    "LockingChainDoor": "rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4",
+    "LockingChainIssue": {
+      "currency": "XRP"
+    },
+    "IssuingChainDoor": "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
+    "IssuingChainIssue": {
+      "currency": "XRP"
+    }
+  }
+}
+```
+
+
+{% include '_snippets/tx-fields-intro.md' %}
+
+| Field         | JSON Type           | [Internal Type][] | Description        |
+|:--------------|:--------------------|:------------------|:-------------------|
+| `XChainBridge`| String | Object | _Required_ The XChainBridge stanza represents the bridge for which the witness is attesting transactions. |
+| `LockingChainDoor` | String | AccountID | The door account on the locking chain. |
+| `LockingChainIssue` | String | Token | The token that is bridged on the locking chain. |
+| `IssuingChainDoor` | String  |  AccountID | The door account on the issuing chain. |
+| `IssuingChainIssue` | String | Token | The token that is bridged on the issuing chain. |
```

Comment:
Given that these are all subfields of XChainBridge, should they just be specified in some sort of XChainBridge page, so it doesn't have to be repeated everywhere? I think we do this with other things too.